### PR TITLE
Replace circle ci

### PR DESCRIPTION
**Pull request by evnu**
This PR replaces CircleCI again with Travis (#96). CircleCI does not check pull requests, maybe Travis CI does it again.

Note that Travis CI does not store build artifacts (beside caching); we would need to upload crash artifacts to S3.

### CODE REVIEW

**Comment by evnu** [comment]
@alfert We originally switched to CircleCI in #96 because Travis did not check pull requests back then. It seems that this has since been fixed; there is a pipeline running for this pull request [here](https://travis-ci.org/alfert/propcheck/builds/594535979?utm_source=github_status&utm_medium=notification).

File: .travis-scripts/verify_storing_counterexamples.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+#
+# Verify that we can configure storing counterexamples
+#
+
+mix test test/store_counter_examples_test.exs --include will_fail:true && exit 1
+mix test test/verify_counter_examples_test.exs
```
Comment (alfert) [inline .travis-scripts/verify_storing_counterexamples.sh]: That's a clever idea to provide a separate build task.

**Comment by alfert** [comment]
@evnu I saw the CI running as well,  good new! Thanks for this PR!

File: .travis-scripts/verify_storing_counterexamples.sh (hunk 1 above)
Comment (evnu) [reply .travis-scripts/verify_storing_counterexamples.sh]: I actually got that from the Travis documentation. :)

**Pull request merged**